arrocke/gloss-translation
# feat: show verse translation

## What has changed



Added a verse translation line above the glossing area.

## Connected Issues



Closes #172 

## QA Steps



1. Select a few bible translations for a language on the manage language page.
2. Go to the translation page for that language.
    - The translation for the given verse should appear at the top.

## Post-Deployment
